test/e2e/performanceprofile/functests/2_performance_update/nodes_test.go

```diff
@@ -0,0 +1,21 @@
+package __performance_update
+
+import (
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/nodes"
+)
+
+//Tests for testing some of the node utils functions.
```

Comment:
I would drop the comment perhaps and change the title to something more meaningful.
Describe - Node utils
Context - CPU tools
Add:  It - GetRanges should return valid CPU/NUMA ranges

Reply:
@yanirq i have updated this on the latest Patch